vendor.conf

```diff
@@ -12,7 +12,7 @@ github.com/cpuguy83/go-md2man v1.0.8
 github.com/davecgh/go-spew 346938d642f2ec3594ed81d874461961cd0faa76 # v1.1.0
 github.com/dgrijalva/jwt-go a2c85815a77d0f951e33ba4db5ae93629a1530af
 github.com/docker/distribution 83389a148052d74ac602f5f1d62f86ff2f3c4aa5
-github.com/docker/docker bd224b5fe564f343e22fa76ae8569ec949f35802
+github.com/docker/docker 0b7cb16dde4a20d024c7be59801d63bcfd18611b
```

Comment:
Can you move the actual vendoring update, and changes to `vendor.conf` to a separate commit?